dev/io.openliberty.webcontainer.servlet.6.0.internal_fat/fat/src/io/openliberty/webcontainer/servlet60/fat/tests/Servlet60CookieSetAttributeTest.java

```diff
@@ -0,0 +1,179 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.webcontainer.servlet60.fat.tests;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.logging.Logger;
+
+import org.apache.hc.client5.http.classic.methods.HttpGet;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpResponse;
+import org.apache.hc.client5.http.impl.classic.HttpClientBuilder;
+import org.apache.hc.core5.http.Header;
+import org.apache.hc.core5.http.io.entity.EntityUtils;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.annotation.Server;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ * Test Cookie.setAttribute
+ *
+ * https://jakarta.ee/specifications/servlet/6.0/apidocs/jakarta.servlet/jakarta/servlet/http/cookie#setAttribute(java.lang.String,java.lang.String)
+ *
+ * Any cookie attribute can be sent with a response. Request does not contain attribute.
+ */
+@RunWith(FATRunner.class)
+@Mode(TestMode.FULL)
+public class Servlet60CookieSetAttributeTest {
+
+    private static final Logger LOG = Logger.getLogger(Servlet60CookieSetAttributeTest.class.getName());
+    private static final String TEST_APP_NAME = "CookieSetAttributeTest";
+
+    //All tests have this response.  Main data to verify are in the Set-Cookie headers.
+    private static final String expectedGeneralResponse = "Hello from the CookieSetAttributeServlet";
+
+    //Reuse existing server since they have same configuration
+    @Server("servlet60_getRealPathTest")
+    public static LibertyServer server;
```

Comment:
I'd really like to keep the 1 java class to 1 server mapping if possible to avoid issues in the future.  It also makes finding logs for a particular test much easier.